packages/interactor/src/constructor.ts

```diff
@@ -190,7 +177,31 @@ export function instantiateInteractor<E extends Element, F extends Filters<E>, A
     });
   }
 
-  return interactor as Interactor<E, FilterParams<E, F>> & ActionMethods<E, A>;
+  return interactor as BaseInteractor<E, FilterParams<E, F>> & ActionMethods<E, A>;
+}
+
+export function instantiateInteractor<E extends Element, F extends Filters<E>, A extends Actions<E>>(
+  options: InteractorOptions<E, F, A>,
+): Interactor<E, FilterParams<E, F>> & ActionMethods<E, A> {
+  let interactor = createInteractor(options, unsafeSyncResolveUnique)
+
+  return Object.assign(interactor, {
+    exists(): ReadonlyInteraction<void> {
+      return check(`${interactor.description} exists`, () => {
+        return converge(() => {
+          resolveNonEmpty(unsafeSyncResolveParent(options), options);
+        });
+      });
+    },
+
+    absent(): ReadonlyInteraction<void> {
+      return check(`${interactor.description} does not exist`, () => {
+        return converge(() => {
+          resolveEmpty(unsafeSyncResolveParent(options), options);
+        });
+      });
+    }
```

Comment:
What about `has` and `is`?

Reply:
They defined in the `createInteractor` function. I can move them here if it makes more sense.